packages/ag-charts-community/src/main-modules.ts

```diff
@@ -1,6 +1,34 @@
-export { AgCharts } from './api/agCharts';
-export { VERSION } from './version';
-export * as _Scene from './integrated-charts-scene';
-export * as _Theme from './integrated-charts-theme';
-export * as _Util from './integrated-charts-util';
-export { registerInbuiltModules as setupCommunityModules } from './chart/factory/registerInbuiltModules';
+import type { IntegratedChartModule, IntegratedSparklineModule } from 'ag-charts-types';
+
+import { AgCharts } from './api/agCharts';
+import { registerInbuiltModules } from './chart/factory/registerInbuiltModules';
+import * as _Scene from './integrated-charts-scene';
+import * as _Theme from './integrated-charts-theme';
+import * as _Util from './integrated-charts-util';
+import { VERSION } from './version';
+
+export const ChartCommunityModule: IntegratedChartModule = {
+    AgCharts,
+    VERSION,
+    // @ts-ignore
+    _Scene,
+    // @ts-ignore
+    _Theme,
+    _Util,
+    create: AgCharts.create,
+    setup: registerInbuiltModules,
+    isEnterprise: false,
+};
+
+export const SparklineModule: IntegratedSparklineModule = {
+    AgCharts,
+    VERSION,
+    // @ts-ignore
+    _Scene,
+    // @ts-ignore
+    _Theme,
+    _Util,
```

Comment:
Sparklines from the grid side of things does not actually require these sections but maybe leave for now.